actors/builtin/miner/miner_actor.go

```diff
@@ -400,11 +400,8 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 		rt.Abortf(exitcode.ErrIllegalArgument, "seal challenge epoch %v must be before now %v", params.SealRandEpoch, rt.CurrEpoch())
 	}
 
-	challengeEarliest := sealChallengeEarliest(rt.CurrEpoch(), params.SealProof)
+	challengeEarliest := rt.CurrEpoch() - ChainFinality - MaxPreCommitRandomnessLookback
```

Comment:
I don't think `ChainFinality` should be part of this. I'm guessing you preserved it from the prior state. @nicola could you please confirm?

Reply:
It will not, we will set `MaxPreCommitRandomnessLookback` in the right way (if it will need finality, we will do so in the param)